app/utilities/firebase/firebaseActions.tsx

```diff
@@ -0,0 +1,81 @@
+import firebase from "./firebase"
+import {GAME_PHASE, QUESTION_KEY, ROOM_KEY, TREE_NODES, USERS_KEY, USER_KEY} from "./firebaseConstants"
+
+export function createRoom () {
+  // ROOT
+  var ref = firebase.database().ref("/")
+  // Save this key some where
+  return ref.push({
+    [ROOM_KEY.GAME_PHASE]: GAME_PHASE.START,
+    [ROOM_KEY.TURN]: 0,
+    [ROOM_KEY.QNA]: "",
+    [ROOM_KEY.CURR_ROOM_SIZE]: 0
+  }).then (snap => {
+    return snap.key;
+  });
+}
+
+export function getCurrentRoomSize (roomKey: string) {
+  var roomRef = firebase.database().ref(`${roomKey}`)
+  return roomRef.get()
```

Comment:
just curiosity, is `get()` different from `once('value', (snapshot) => ...` ?

Reply:
i think once() gets executed when the db changes, (child most likely). Whereas get is just called instantly when the method is called